src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1812,10 +1812,8 @@ namespace winrt::TerminalApp::implementation
                 auto unopenedUri = FindName(L"UnopenedUri").try_as<Windows::UI::Xaml::Documents::Run>();
 
                 // Insert the URI
-                const auto uriMsg = fmt::format(std::wstring_view{ RS_(L"UnopenedUriText") },
-                                                eventArgs.Uri().c_str());
-                unopenedUri.Text(uriMsg);
-                
+                unopenedUri.Text(eventArgs.Uri().c_str());
+
```

Comment:
And for our last master stroke, we can get rid of c_str since we are just moving hstrings bac and forth :)

Reply:
(Avoid cstr when possible with these smart types! It’s like getting the raw string out using an escape hatch... and in this case, you are going and putting it right back into an hstring!)